wrappers/csharp/cs-tutorial-8-D400-on-chip-calibration/Window.xaml.cs

```diff
@@ -57,13 +57,14 @@ public CaptureWindow()
                 using (var ctx = new Context())
                 {
                     var devices = ctx.QueryDevices();
-                    if ( 0 == devices.Count )
+                    var dev = devices[0];
+
+                    if ( 1 != devices.Count || (!ExampleAutocalibrateDevice.IsTheDeviceD400Series(dev)) )
```

Comment:
Why not just ask for D400-series cameras in QueryDevices?

Reply:
The QueryDevices for C# currently doesn't support masking product lines.
Changing it now may affect other demos and therefore should be handled separately.